lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendarSelectionConfigurationConstants.m

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "RCTBPKCalendarSelectionConfigurationConstants.h"
+
+#import <Backpack/Calendar.h>
+
+@implementation RCTBPKCalendarSelectionConfigurationConstants
+
++ (BPKCalendarSelectionConfiguration *)single {
+    return [BPKCalendarSelectionConfigurationSingle new];
```

Comment:
As react only sends changes in props, we should find that new instances are only changed when the selection type prop changes